Feature/georeferencing (#330)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcGeographicCRS/Documentation.md

> _IfcGeographicCRS_ is a coordinate reference system that is based on a ellipsoid. Geographic CRS typically requires two angles and a height to determine a unique position.

Reviewer: A geographic coordinate reference system is a CRS that uses a three-dimensional spherical surface to determine locations on the Earth. Any location on Earth can be referenced by a point with longitude and latitude coordinates.

> _IfcGeographicCRS_ is a coordinate reference system that is based on a ellipsoid. Geographic CRS typically requires two angles and a height to determine a unique position.

Reviewer: ```suggestion
A _IfcGeographicCRS_ is a CRS that uses a three-dimensional ellipsoid surface to determine locations on the Earth. Any location on Earth can be referenced by a point with longitude and latitude coordinates.
```

Reviewer: ![grafik](https://user-images.githubusercontent.com/73635642/148844629-157b066f-1f12-4fca-ae17-effd140c33ef.png)

> A _IfcGeographicCRS_ is a CRS that uses a three-dimensional ellipsoid surface to determine locations on the Earth. Any location on Earth can be referenced by a point with longitude and latitude coordinates.

Reviewer 2: ```suggestion
A _IfcGeographicCRS_ is a CRS that uses a three-dimensional ellipsoid surface to determine locations on the Earth. Any location on Earth can be referenced by a point with longitude and latitude coordinates and ellipsoidal height.
```

> _IfcGeographicCRS_ is a coordinate reference system that is based on a geoid or ellipsoid. Geographic CRS typically requires two angles and a height (can be derived) to specify a location on a body such as ellipsoid, geoid etc

Reviewer 2: Please delete geoid. This establishes (normal) height and uses an ellipsoid as base for localization. It is not part of / type of a *geographic* CRS.

Geographic CRS's height is "height above/below the ellipsoid's surface".

> A _IfcGeographicCRS_ is a CRS that uses a three-dimensional ellipsoid surface to determine locations on the Earth. Any location on Earth can be referenced by a point with longitude and latitude coordinates and the height above the ellipsoid.

Reviewer 2: ```suggestion
A _IfcGeographicCRS_ is a CRS that uses a three-dimensional ellipsoid surface to determine locations on the Earth. Any location on Earth can be referenced by a point with longitude and latitude coordinates and the height above/below the ellipsoid.
```

> _IfcGeographicCRS_ is a coordinate reference system that is based on a ellipsoid. Geographic CRS typically requires two angles and a height to determine a unique position.

Reviewer 2: ```suggestion
_IfcGeographicCRS_ is a coordinate reference system that is based on an ellipsoid. Geographic CRS typically requires two angles and a height above the ellipsoid's surface to determine a unique position in 3D space.
```
